ee/debug/checkups/enroll-secret.go

```diff
@@ -26,6 +26,18 @@ func (c *enrollSecretCheckup) Run(_ context.Context, extraFH io.Writer) error {
 	secretStatus := make(map[string]Status, 0)
 	secretSummary := make(map[string]string, 0)
 
+	// Check registrations first -- will only be available if we're running in situ
+	if registrations, err := c.k.Registrations(); err == nil {
+		for i, registration := range registrations {
```

Comment:
Should we log something if there are no registrations?

Reply:
Updated!